res/default_ddoc_theme.ddoc

```diff
@@ -2,6 +2,8 @@ LPAREN = (
 RPAREN = )
 BACKTICK = `
 DOLLAR = $
+LF =
+$(LF)
```

Comment:
Glad you asked. A macro present with no arguments in its own definition expands to nothing, so this makes sure there's an empty line inside the macro. Without that, newlines are removed so the macro would not contain a newline.

Similarly, one space is surprisingly subtle to define: `SPACE = $(SPACE) $(SPACE)`.